include/atomics/receiver_cadmium.hpp

```diff
@@ -25,48 +35,82 @@
 
 #include "../data_structures/message.hpp"
 
+/** 
+* namespace is used to organize code into logical groups and to prevent name collisions.
+*/
 using namespace cadmium;
 using namespace std;
 
-//Port definition
+/**
+* Structure consists of class public output and input Messages
+* Port definition 
+*/
     struct receiver_defs {
 		struct out : public out_port<message_t> {
 		};
 		struct in : public in_port<message_t> {
 		};
 	};
    
+
     template<typename TIME>
+    
     class Receiver {
-        using defs = receiver_defs; // putting definitions in context
+
+        
+        using defs = receiver_defs;
+        /** 
+        * Putting reciever definitions in context. 
+        */  
         public:
-            //Parameters to be overwriten when instantiating the atomic model
+        
+        /** 
+        * Time constant Parameter
+        */
             TIME	PREPARATION_TIME;
```

Comment:
You need a brief description as follow:
 TIME	PREPARATION_TIME;    /**< description */ //!< Brief description after the member

Reply:
not resloved